tiles-generation/cdk/README.md

```diff
@@ -0,0 +1,56 @@
+# Tiles generation pipeline with CDK
+
+ This project in `tilesGenerationPipeline` folder helps to build a raster map tiles generaion pipeline on Amazon ECS in AWS CDK. The docker base image is come from https://github.com/Overv/openstreetmap-tile-server
+.
+#  Getting Started
+
+## Prerequisites
+Install AWS CLI and Node.js first.
```

Comment:
A link to how to install will be helpful

Reply:
Sure, will update.